var/spack/repos/builtin/packages/py-mpmath/package.py

```diff
@@ -28,6 +28,7 @@
 class PyMpmath(PythonPackage):
     """A Python library for arbitrary-precision floating-point arithmetic."""
     homepage = "http://mpmath.org"
-    url      = "https://pypi.io/packages/source/m/mpmath/mpmath-all-0.19.tar.gz"
```

Comment:
Is 0.19 still available for download? It looks like the URL changed (dropped the `all`).

Reply:
Hmm, I could download it manually before, but now using Spack I noticed that the checksum of 0.19 changed. The old `url` is still available. Do you prefer to use a `url=` parameter, or change the checksum in the 0.19 version?

Reply:
I would prefer to change the checksum. Makes the package simpler. Hopefully there isn't much of a difference.